{"name": "test_login", "status": "failed", "statusDetails": {"message": "Failed: Fixture \"app\" called directly. Fixtures are not meant to be called directly,\nbut are created automatically when test functions request them as parameters.\nSee https://docs.pytest.org/en/latest/fixture.html for more information about fixtures, and\nhttps://docs.pytest.org/en/latest/deprecations.html#calling-fixtures-directly about how to update your code.", "trace": "Fixture \"app\" called directly. Fixtures are not meant to be called directly,\nbut are created automatically when test functions request them as parameters.\nSee https://docs.pytest.org/en/latest/fixture.html for more information about fixtures, and\nhttps://docs.pytest.org/en/latest/deprecations.html#calling-fixtures-directly about how to update your code."}, "attachments": [{"name": "stdout", "source": "213604cb-849f-4632-8f1d-f86d5aa0b72e-attachment.txt", "type": "text/plain"}], "start": 1559919276300, "stop": 1559919276300, "uuid": "31b4631b-1cc6-4875-90d3-68cf119c93e3", "historyId": "1d6c5e711b29cf7bad43b6e24d6adc90", "fullName": "tests.test#test_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test"}, {"name": "host", "value": "Work-Mac-Mini.local"}, {"name": "thread", "value": "90455-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test"}]}